export/policy.ts

```diff
@@ -0,0 +1,21 @@
+import { PhylumApi } from 'phylum';
+
+/**
+ * Given a job ID and a policy, evaluate the job against the policy.
+ */
+export async function evaluatePolicy(jobId: string, policy: string): Promise<any> {
+    const data = JSON.stringify({"ignoredPackages":[], "policy": {}});
+
```

Comment:
```suggestion
```

Reply:
This might explain the issue you were seeing sending `{}`. I meant if the entire body is `{}` it should work the same as with no body. `{"policy": {}}`, however, is invalid because the policy property is a nullable string.